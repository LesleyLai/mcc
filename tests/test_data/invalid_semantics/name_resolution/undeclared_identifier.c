int main(void)
{
  return x;
}
