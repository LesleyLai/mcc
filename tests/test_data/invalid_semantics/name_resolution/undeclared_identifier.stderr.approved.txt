{{filename}}:3:10: Error: use of undeclared identifier 'x'
3 |   return x;
  |          ^

